scheduler/src/test/java/com/flipkart/flux/redriver/scheduler/RedriverServiceTest.java

```diff
@@ -91,19 +88,6 @@ public void testLargeNoOfRedriveMessages_allShouldBeRedrived() throws Exception
         }
     }
 
-    @Test
-    public void testNoRedriveMessages_shouldNotRedriveMessagesWithScheduledTimeOfFuture() throws Exception {
-        long now = System.currentTimeMillis();
-        when(messageManagerService.retrieveOldest(0, batchSize)).
-                thenReturn(Arrays.asList(new ScheduledMessage(1l, "sample-state-machine-uuid", now + 9000),
-                        new ScheduledMessage(2l, "sample-state-machine-uuid", now + 10000)));
-
-        redriverService.start();
-
-        Thread.sleep(1000);
-        verifyZeroInteractions(redriverRegistry);
-    }
-
```

Comment:
Why is this test case removed ?

Reply:
will add a integration test to cover this unit one.